### ISSUE TITLE
Generate fixture data using props library

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#243]
**Is your feature request related to a problem? Please describe.**
The fixture data for a component's tests and stories is manually curated. This is unmanageable and a duplication of effort.

**Describe the solution you'd like**
Use of a library to generate fixture data for components' tests and stories. Use: 
- https://github.com/typicode/react-fake-props or
- https://github.com/markalfred/react-generate-props

The fixture data should have the ability to output markdown too.

It should be clear in the description as to what library was chosen and why.

Testing : Its more of a developer experience so no testing effort required.

**Comment by ryanmccombe** [comment]
**_TLDR: I don't think we should do this_**

WIP PR here to get a sense of what this might look like: https://github.com/bbc/simorgh/pull/1998

Not at all sure that generating props automatically based on the `propTypes` is a good idea - it makes our snapshots less representative of the realistic use of these components

We need to provide sample props to demo the components in storybook anyway - might it be a better idea to export those sample props, and re-use them in the snapshot tests?

There's an example of what that might look like here: https://github.com/bbc/simorgh/pull/1998/commits/f2a23beb79d139ad060463bf9e6461b69a35f09a

Snapshots / stories using the same props may lead to a better developer / QA experience too - when a snapshot fails, you can use storybook to visually check the component using the exact same props from the failing snapshot.

**Comment by jamesbhobbs** [comment]
See comment above re why. Thanks @ryanmccombe
